sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -242,7 +242,7 @@ func (host *nodeLanguageHost) GetRequiredPlugins(ctx context.Context,
 	// minor version, we will issue a warning to the user.
 	pulumiPackagePathToVersionMap := make(map[string]semver.Version)
 	plugins, err := getPluginsFromDir(
-		req.GetProgram(),
+		filepath.Join(req.Pwd, req.Program),
```

Comment:
I don't think so.
Program is a relative path from the project root to the program, and we normally start language plugins with their working directory set to the project root.
This now just does that explicitly because when we're doing matrix testing the plugins working directory is the test directory.
We have quite a few tests for odd combinations of this in the integration tests and none of those flagged this as an issue.